psk_fitness/psk_fitness/Repositories/ExerciseRepository.cs

```diff
@@ -69,5 +68,30 @@ public async Task<ExerciseDisplayDTO> DeleteExerciseAsync(int id)
             }
             return mapper.Map<ExerciseDisplayDTO>(exercise);
         }
+
+        public async Task<List<ExerciseDisplayDTO>> GetExercisesForUser(string userId)
```

Comment:
GetExercisesForUser uses userId and GetExercisesForWorkout uses userEmail for user identification? What's the motive?